src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -106,13 +108,18 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   override def adjustInputPaths(call: Call, inputs: CallInputs): CallInputs = inputs map { case (k, v) => (k, gcsPathToLocal(v)) }
   override def adjustOutputPaths(call: Call, outputs: CallOutputs): CallOutputs = outputs map { case (k,v) => (k, gcsPathToLocal(v)) }
 
+  private def makeLogger(descriptor: WorkflowDescriptor) = {
+    WorkflowLogger(descriptor, otherLoggers=Seq(LoggerFactory.getLogger(getClass.getName)))
+  }
+
   private def writeAuthenticationFile(workflow: WorkflowDescriptor) = authenticated { connection =>
     val path = GoogleCloudStoragePath(gcsAuthFilePath(workflow))
+    val log = makeLogger(workflow)
 
     generateAuthJson(jesConf.dockerCredentials, getGcsAuthInformation(workflow)) foreach { content =>
       def upload(prev: Option[Unit]) = connection.storage.uploadJson(path, content)
 
-      logger.info(s"Creating authentication file for workflow ${workflow.id} at \n ${path.toString}")
```

Comment:
Yeah, I looked into removing the one use of `LazyLogging` in this file and it's actually annoyingly more complicated than one might guess... The function that uses the lazy logger is called *in the constructor of `WorkflowDescriptor`* so we can't use a full `WorkflowLogger` because we don't have a `WorkflowDescriptor` yet.  Yeah it's annoying though I don't mind leaving it as is for now.